docs/blog/posts/how_to_uncompress_large_files.md

```diff
@@ -0,0 +1,71 @@
+---
+date:
+  created: 2023-08-18
+categories:
+    - zip
+authors:
+    - nrapstin
+---
+
+
+Compressed files are widely used to save space and facilitate file transfers. Understanding how to uncompress these files ensures quick access to your data, no matter the format. This post will guide you through the process of uncompressing files on the Yens.
+
+
+<!-- more -->
+
+# How do I extract compressed files?
+
+
+Large collections of files will often be compressed in order to make them more friendly for transfer or long-term storage. Below are examples on how to uncompress a few common compression file types on the Yens.
```

Comment:
This paragraph can be combined with the previous one. They are saying the same thing.